backend/api/urls.py

```diff
@@ -25,4 +25,5 @@
     path("api/posts/<int:pk>/", views.PostDetail.as_view(), name="post-detail"),
     path("api/posts/delete/<int:pk>/", views.PostDelete.as_view(), name="post-delete"),
     path("api/signup", views.SignUpView.as_view(), name="signup-view"),
+    path("api/v2/auth/", include("authentication.urls")),
```

Comment:
@maunga-et let's not change the api version yet. I recommend we name the JWT endpoints something like `api/token/`. Then we can have endpoints like:

- `api/token/` for generating a token keypair or login
- `api/token/refresh/` to generate new refresh tokens and
- `api/token/logout` to invalidate & blacklist tokens and then log the user out.